server/items/itemModel.js

```diff
@@ -13,6 +13,16 @@ var ItemSchema = new mongoose.Schema ({
   itemLocation :{
     type: Object,
     required: true
+  },
+
+   itemLng: {
```

Comment:
Delete is looking great guys! Quick question: why do we need to create new lng/lat if we have it stored as an obj in itemLocation?